config/routes.rb

```diff
@@ -20,7 +20,7 @@
   end
 
   namespace :ncr do
-    resources :proposals
+    resources :work_orders
   end
```

Comment:
Should we worry about redirecting from the old URLs? My guess is no, but wanted to bring it up.

/cc @phirefly